lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -27,12 +36,36 @@ $badge-min-height: 20px;
 		@include label-xs-default-bold-uppercase;
 	}
 
+	&__icon {
+		align-items: center;
+		display: flex;
+		height: $icon-xxs;
+		margin-right: $space-xxs;
+		width: $icon-xxs;
```

Comment:
why do we have to override sizes defined by Icon component?

Reply:
True, that's unnecessary